src/neo4j-arc/cypher-language-support/index.ts

```diff
@@ -26,6 +26,12 @@ export {
   initalizeCypherSupport,
   getText
 } from './cypher-editor/editorSupport'
+export {
+  parse,
```

Comment:
parse is not really a describing name, maybe rename to something more descriptive?